tls/pk.c

```diff
@@ -259,6 +259,7 @@ ttls_pk_free(TlsPkCtx *ctx)
 	if (ctx->pk_info->ctx_free_func)
 		ctx->pk_info->ctx_free_func(ctx->pk_ctx);
 
+	ctx->pk_info = NULL;
```

Comment:
It seems this is required to not to fail https://github.com/tempesta-tech/tempesta/blob/MekhanikEvgenii/fix-1888/tls/pkparse.c#L679 , but basically we should not use `pk` after freeing it, so I propose to just call `ttls_pk_init()` just after https://github.com/tempesta-tech/tempesta/blob/MekhanikEvgenii/fix-1888/tls/pkparse.c#L676